src/index.js

```diff
@@ -0,0 +1,197 @@
+        lon: lon,
+      },
+    })
+    .then((response) => {
+      console.log(response.data);
+      return response.data.current.temp;
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const handleCityEntered = (event) => {
+  console.log(`City entered ${event}`);
+  const inputCityName = event.target.value;
+  const displayCityName = document.getElementById('cityDisplay');
+  state.city = inputCityName;
+  displayCityName.textContent = state.city;
+};
+
+const increaseTemp = (event) => {
+  state.temperature += 1;
+  colorChange();
+};
+
+const decreaseTemp = (event) => {
+  state.temperature -= 1;
+  colorChange();
+};
+
+const reset = (event) => {
+  // state.temperature = 79; //change it after API
+  state.city = 'London';
+  document.getElementById('city').value = state.city;
+  document.getElementById('cityDisplay').textContent = state.city;
+
+  go(event);
+};
+
+const go = (event) => {
+  event.preventDefault()
+  console.log(`Getting lat/lon and weather for city ${state.city}`);
+  getLatAndLon(state.city)
+    .then((latLon) => getWeatherKelvin(latLon.lat, latLon.lon))
+    .then((temp) => {
+      state.temperature = state.isF ? convertKtoF(temp) : convertKtoC(temp);
+      colorChange();
+    });
+};
+
+const changeSky = (event) => {
+  const inputSky = document.getElementById('skySelect');
+
+  const bg = document.getElementsByTagName('body')[0];
+
+  switch (inputSky.value) {
+    case 'Lover':
+      bg.style.background = 'url("assets/lover.jpeg")';
+      break;
+    case 'rep':
+      bg.style.background = 'url("assets/rep backup.jpeg")';
+      break;
+    case '1989':
+      bg.style.background = 'url("assets/1989 sky.jpeg")';
+      break;
+    case 'Red':
+      bg.style.background = 'url("assets/red.jpeg")';
+      break;
+    case 'Speak Now':
+      bg.style.background = 'url("assets/speak now.jpeg")';
+      break;
+    case 'fearless':
+      bg.style.background = 'url("assets/fearless.jpeg")';
+      break;
+  }
+
+  bg.style.backgroundSize = 'cover';
+};
+
+const colorChange = () => {
+  let tempColor;
+  let currentOutfit;
+  let lyric;
+
+  let temp = state.temperature;
+  if (!state.isF) {
+    temp = convertCtoF(temp);
+  }
+
+  if (temp >= 80) {
+    tempColor = 'red';
+    currentOutfit = 'assets/80.jpeg';
+    lyric = 'Shake it Off';
+  } else if (temp >= 70) {
+    tempColor = 'orange';
+    currentOutfit = 'assets/75.jpeg';
+    lyric = 'It feels like a perfect night to dress up like hipsters';
+  } else if (temp >= 60) {
+    tempColor = 'yellow';
+    currentOutfit = 'assets/50.jpeg';
+    lyric = 'Autumn leaves falling down like pieces into place';
+  } else if (temp >= 50) {
+    tempColor = 'green';
+    currentOutfit = 'assets/60.jpg';
+    lyric = 'There is something about the rain';
+  } else {
+    tempColor = 'teal';
+    currentOutfit = 'assets/49.jpeg';
+    lyric = 'Forever Winter if you go';
+  }
```

Comment:
Like I mentioned in your `changeSky` method, you can use a list here to contain objects that have the color, outfit and lyric values for each temperature range. Then your method would iterate through the constant list and find the first record that has an upper bound higher than our temperature, then use it as the source of picking the color/outfit/lyric.